R/print_method.R

```diff
@@ -32,12 +33,12 @@ tbl_format_header.tidySummarizedExperiment <- function(x, setup, ...) {
                 named_header
             ) %>%
             # Add further info single-cell
-            append(sprintf(
-                "\033[90m Features=%s | Samples=%s | Assays=%s\033[39m",
+            append( cli::col_br_black( sprintf(
+                " Features=%s | Samples=%s | Assays=",
                 number_of_features,
                 number_of_samples,
                 assay_names %>% paste(collapse=", ")
-            ), after = 1)
+            )), after = 1)
```

Comment:
Great, this would be amazing in another PR. This is quite fundamental. And I want to replicate this in all tidy transcriptomics.  But I don't want to have you too much work